player.py

```diff
@@ -122,24 +123,24 @@ def setFrameData(self, lines, score, level, score_label, level_label): # lines,
 
 		if self.pending_lines:
 			changed = True
-			self.pending_lines = False;
+			self.pending_lines = False
 
 			if lines == None or lines == 0:
 				self.tetris_line_count = 0
 			else:
 				cleared = lines - (self.lines or 0)
 				if cleared == 4:
-					self.tetris_line_count =+ 4
+					self.tetris_line_count += 4
```

Comment:
This bug is rather awesome in a way 😅  Normally a typo of flipping 2 characters in code causes a syntax error, but in this case, the result was still valid python LOL